mmv1/third_party/terraform/services/container/resource_container_cluster.go.tmpl

```diff
@@ -2386,6 +2386,27 @@ func ResourceContainerCluster() *schema.Resource {
 					},
 				},
 			},
+			"rbac_binding_config": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Computed:    true,
+				MaxItems:    1,
+				Description: `RBACBindingConfig allows user to restrict ClusterRoleBindings an RoleBindings that can be created.`,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"enable_insecure_binding_system_unauthenticated": {
+							Type:              schema.TypeBool,
+							Optional:          true,
+							Description:       `Setting this to true will allow any ClusterRoleBinding and RoleBinding with subjects system:anonymous or system:unauthenticated.`,
```

Comment:
Should this have `AtLeastOneOf` set for the two enable_* fields? similar to protect_config: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/container/resource_container_cluster.go.tmpl#L1221

Reply:
Unless there is a difference between 
`rbac_binding_config {}`
and
`rbac_binding_config { false, false }`
in the API

Reply:
ill check but im not sure

Reply:
We don't need atLeastOneOf, since the default function will handle the case, we allow `rbac_binding_config {}` for internal refer google3/cloud/kubernetes/server/patch/field/security/cluster_rbac_binding_config.go